pytket/setup.py

```diff
@@ -254,6 +254,7 @@ def finalize_options(self):
         "jinja2 ~= 3.0",
         "types-pkg_resources",
         "typing-extensions ~= 4.2",
+        "qwasm ~= 1.0",
```

Comment:
Is qwasm open-source?

Reply:
Yes:
https://github.com/CQCL/qwasm
https://pypi.org/project/qwasm/

Reply:
Cool, would be good to add a link to the repo from the pypi page.